### PR TITLE
fix(MSA-1185):[USP] Customers/ Document upload: Not user-friendly error returned when document uploaded is too large

### PR DESCRIPTION
**Pull request by iammohanselvam**
…or returned when document uploaded is too large

### CODE REVIEW

**Review by BozhidarTodorov — APPROVED** [review]

**Comment by stoberov** [comment]
@iammohanselvam - pls, check my last comment from a few days ago.
@BozhidarTodorov - please, add your review.

**Review by stoberov — CHANGES_REQUESTED** [review]

File: components/DocumentUpload/index.js [hunk 1]
```diff
@@ -304,6 +304,14 @@ export default class DocumentUpload extends Component {
         this.setState({
             isUploading: false
         });
+        // fix(MSA-1185): Document upload: Show user friendly error message when document uploaded is too large
+        if (attachmentResult.error) {
+            let errorMsg = attachmentResult.error.message || attachmentResult.error.statusText;
+            if (attachmentResult.error.body?.error === 'Request Entity Too Large') {
```
[the remark this answers is not in the document]
Comment (stoberov) [reply components/DocumentUpload/index.js:310]: Абсолютно съм съгласен, бях оставил същия коментар, сега виждам, че не го сейвнало. 
Преведено на английски за Мохан и цъкам Resolve тук :D

[the remark this answers is not in the document]
Comment (iammohanselvam) [reply components/DocumentUpload/index.js:310]: Got it, working on it (y)

[the remark this answers is not in the document]
Comment (iammohanselvam) [reply components/DocumentUpload/index.js]: Hi @stoberov  please verify now.

**Review by BozhidarTodorov — APPROVED** [review]

File: components/DocumentUpload/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (stoberov) [reply components/DocumentUpload/index.js]: Lookin' good!

**Pull request merged**